doc/ref_impl/cntt-ri/chapters/chapter04.md

```diff
@@ -7,60 +7,68 @@
 * [4.1 Introduction](#4.1)
 * [4.2 Lab Requirement](#4.2)
 * [4.3 Lab Topology](#4.3)
-* [4.4 Lab HW Spec](#4.4)
-* [4.5 Lab Use Guidelines](#4.5)
+* [4.4 Lab Use Guidelines](#4.4)
 
 
 <a name="4.1"></a>
-## 4.1 Introdcution
+## 4.1 Introduction
+This chapter provides the laboratory hardware requirements needed to deploy the reference implementation.  These requirements represent a minimum set of requirements, where different labs may have hardware or capabilities exceeding these minimum specifications.  
 
+The lab provides a set of bare-metal servers, supported by a dedicated network infrastructure.  Users should be provided access to the servers, their operating systems, and out-of-band management (i.e. IPMI, IDRAC, ILO, etc.).  
 
 <a name="4.2"></a>
-## 4.2 Lab Requirement
-
-- Controller Nodes:
-  - 3 x
-    - 2x dual-port 10Gbps NIC.
-    - 2.2GHz 14C/28T.
-    - 256GB RAM.
-    - 10TB HDD.
-    - 3.2TB SSD.
-- Compute Node
-  - 10x
-    - 2x dual-port 25Gbps NIC.
-    - 2.2GHz 24C/48T.
-    - 512GB RAM.
-    - 2TB HDD
-    - Supports 2x Nodes for B/N/C flavors
-- Jump/Baremetal manager
-  - 1x
-    - 2x dual-port 10Gbps NIC.
-    - 2.2GHz 14C/28T.
-    - 256GB RAM.
-    - 10TB HDD
-- Networking
-  - 1x Spine Switch
-    - Total: 32x100G
-  - 2x Leaf Switch
-    - 48 x 25/10G
-    - 6 x 100G
-- 1x 48u Rack
-- Cables and Transcievers
-
-<a name="4.3"></a>
-## 4.3 Lab Topology
+## 4.2 Lab Requirements
 
+Labs are organized into one or more *Pods*, where the *Pod* provides the complete set of resources [servers and network(s)] necessary to support the reference implementation installation and operation.  
 
-<a name="4.4"></a>
-## 4.4 Lab HW Spec
+A CNTT compliant *Pod* shall meet the following requirements.
+
+1. One (1) bare-metal server dedicated as a Jump / Test Host
```

Comment:
Where will the "Foundation" node be hosted - e.g. the cloud builder (Director/Undercloud in the case of Red Hat)?

Reply:
Hi Ian, That's an installer-specific question. So far we've been using Airship (1.x), which does not require a dedicated node for the purpose, it uses any (i.e. you pick it) of the future control nodes to perform that function. In the case of RH (OOO), my understanding is the foundation node is ephemeral, so are you thinking it requires a dedicated node?, or it could just as easily be a VM on the jump host. Either way, that's a decision that the person implementing the particular RI would need to make.

Reply:
@markshostak @iangardner22 , for RH tripleO , The undercloud node, it can be VM based on any serevr as it is just used for deployment ( Just :  8-core , with minimum 16 GB of RAM and 100GB available disk)

Reply:
@markshostak @liyingyjy , my recomedation not to mention Bare Metal , to be able to utilize the servers for any VM deployment ( undercloud , Tools ,..)

Reply:
Replaced Bare Metal w/ Physical

Reply:
@markshostak @ASawwaf - points noted and yes I am happy for it to be a VM - just may be we name the "jumphost" something like "utility host" and then we can use it for anything including jump, Director, Puppet-master, Ansible etc...